### PR TITLE
Add privacy info text, visitor tracker, small corrections in templates

### PR DESCRIPTION
**Pull request by akochari**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description

Added a page with privacy info and matomo visitor tracker code. In addition, made some small changes to the templates, making them a bit better organized.

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [x] New feature
- [ ] Documentation update

## Checklist

_If you're unsure about any of the items below, don't hesitate to ask. We're here to help! 
This is simply a reminder of what we are going to look for before merging your code._

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [x] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [ ] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [x] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -45,6 +45,22 @@
...
+                        </button>
+                        <ul class="dropdown-menu">
+                          <li><a class="dropdown-item" href="{% url 'password_change' %}"><i class="bi bi-key me-1"></i>Change Password</a></li>
+                          <li><a class="dropdown-item" href="{% url 'logout' %}"><i class="bi bi-box-arrow-right me-1"></i>Sign out</a></li>
+                        </ul>
+                      </div>
+                    </li>
+
+                    {% else %}
+
```
Comment (sandstromviktor) [inline templates/base.html:63]: Is there a reason for removing the navbar as a separate file? I would prefer to have navbar and footer in separate files and `{% include %}` them. 
Also it is a bit superfluous to use `nav-item` if we're not using the bootstrap `navbar` class. 

_Navbars require a wrapping .navbar with .navbar-expand{-sm|-md|-lg|-xl} for responsive collapsing and color scheme classes._ See [link](https://getbootstrap.com/docs/4.0/components/navbar/)

Looks and works great though.

File: templates/base.html [hunk 2]
```diff
@@ -224,6 +237,22 @@ <h1 class="modal-title fs-5" id="modalConfirmDeleteLabel"></h1>
...
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        (function () {
+            var u = "//matomo.dckube.scilifelab.se/";
+            _paq.push(['setTrackerUrl', u + 'matomo.php']);
+            _paq.push(['setSiteId', '5']);
+            var d = document, g = d.createElement('script'), s = d.getElementsByTagName('script')[0];
+            g.async = true; g.src = u + 'matomo.js'; s.parentNode.insertBefore(g, s);
+        })();
+    </script>
```
Comment (sandstromviktor) [inline templates/base.html:255]: Don't know how to verify that this works, so i'll just approve and hope for the best!

**Review by sandstromviktor — APPROVED** [review]
Good job Arnold!

File: templates/base.html (hunk 1 above)
Comment (akochari) [reply templates/base.html]: Thanks for your comments!
The footer and navbar files that I removed were not ours. I now moved our own navbar and footer out.

Regarding the `nav-item`: that's under the [.nav](https://getbootstrap.com/docs/5.3/components/navs-tabs/#base-nav).
We might want to make the navigation collapsable and use `navbar` instead but I will leave this decision for later.

**Pull request merged**